src/app.js

```diff
@@ -3,12 +3,16 @@ import fs from 'fs/promises';
 import ejs from 'ejs';
 import { loadMovie, loadMovies } from './movies.js';
 import { renderMarkdown } from './markdown.js';
+import fetch from 'node-fetch';
+import { sendReview } from './sendReview.js';
 
 const app = express();
 app.set('view engine', 'ejs');
 app.set('views', './views');
 
 app.use('/static', express.static('./static'));
+app.use(express.urlencoded({ extended: true }));
```

Comment:
Is this actually needed? Maybe I'm missing something, but I don't think you're sending any form data. In review.js, which sends the data to the API, you're using JSON to encode the data, which means that there is no need for "urlencoded forms", which is the default way that browsers send form data when there is no frontend programming involved.

Sending JSON is better IMO, so I think your solution is great. But it does mean that it should be possible to remove this line of code.